test/benchmark/shim/utilities.bench.js

```diff
@@ -41,13 +41,6 @@ suite.add({
   }
 })
 
-suite.add({
```

Comment:
Why was this removed?

Reply:
@bizob2828 removed this method from the agent's shim interface with [this commit](https://github.com/newrelic/node-newrelic/commit/c470ff4fd223ba33a9c067463746b83918edd60d) in 2021; I documented that in the ticket comments. `events.EventEmitter.listenerCount` has apparently been deprecated from Node. Since this was removed from the agent code, we shouldn't be testing on it.